src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -318,14 +324,14 @@ public int RenameTrackingPreview
 
         public int ShowKeywords
         {
-            get { return GetBooleanOption(CompletionOptions.IncludeKeywords); }
-            set { SetBooleanOption(CompletionOptions.IncludeKeywords, value); }
+            get { return 0; }
```

Comment:
We discussed this in the teamroom.  Basically, we can't envision a case where it's beneficial to have completion but not have keywords in it.  

1. You won't get the keywords off of htings like <dot>
2. If the completion list comes up on things like space and identifiers, then it's really important to have keywords in it.
3. The keywords provided are already extremely accurate.  So it's not like random lists have tons of keywords on it that look innapropriate for their locations.

So having this option isn't useful.